src/table.js

```diff
@@ -604,44 +645,36 @@ Table.prototype.delete = function(gaxOptions, callback) {
  *   }
  * };
  *
- * table.deleteRows({
- *   prefix: 'alincoln'
- * }, callback);
- *
- * //-
- * // If you choose to omit the prefix, all rows in the table will be deleted.
- * //-
- * table.deleteRows(callback);
+ * table.deleteRows('alincoln', callback);
  *
  * //-
  * // If the callback is omitted, we'll return a Promise.
  * //-
- * table.deleteRows().then(function(data) {
+ * table.deleteRows('alincoln').then(function(data) {
  *   const apiResponse = data[0];
  * });
  */
-Table.prototype.deleteRows = function(options, callback) {
-  if (is.function(options)) {
-    callback = options;
-    options = {};
+Table.prototype.deleteRows = function(prefix, gaxOptions, callback) {
+  if (is.function(gaxOptions)) {
+    callback = gaxOptions;
+    gaxOptions = {};
+  }
+
+  if (!prefix || typeof prefix === 'function') {
```

Comment:
To be consistent, we use the `is` module:

```js
is.fn(prefix)
```

Reply:
good call. Switched